source/_integrations/slow_pwm.markdown

```diff
@@ -0,0 +1,114 @@
+---
+title: Slow PWM
+description: Convert one or multiple switches to a numeric input using slow pulse width modulation
+ha_category:
+  - Number
+ha_release: '2023.05'
+ha_config_flow: true
+ha_iot_class: Local Polling
+ha_codeowners:
+  - '@antonverburg'
+ha_domain: slow_pwm
+ha_platforms:
+  - number
+ha_integration_type: integration
+---
+
+The `slow_pwm` platform is a `number`. It can be used to control one or multiple digital switches in an analog control algorithm. 
```

Comment:
We don't allow the creation of new platform integrations anymore.

Suggested: The SlowPWM integration can be used to control...